src/mobile/FooterLinks/FooterLinks.scss

```diff
@@ -0,0 +1,53 @@
+@import "../../scss/variables.scss";
+
+.footerLinks {
+  display: flex;
+  flex-wrap: wrap;
+
+  .socialMedia {
+    max-width: 370px;
+    width: 100%;
+    display: flex;
+    gap:20px;
+    padding-bottom: 20px;
+
+    .socialMediaItem {
+      width: 47px;
+      height: 47px;
+      transition: .3s;
+
+      &:hover {
+        filter: drop-shadow(0px 0px 3px $tertiaryColorMain);
```

Comment:
Changing "filter" property has bug in Safari. Test it plz.
Adding `will-change: filter` to `.socialMediaItem` will fix it.